ui/widget-expressions-variables.md

```diff
@@ -166,49 +185,49 @@ Object expression can also be used to simulate a `switch` control statement.
 
 The most common flow control statement in expressions is the [conditional (ternary) operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator), which is very efficient for selecting from two options based on a single boolean criterion.
 If you have a list of possible options, you can combine multiple ternary operators together, but this grows cumbersome very quickly.
-For example, if there is an HVAC with a mode Item that can be set to `heat`, `cool`, `auto`, and `off` modes, it requires 4 nested ternary operators to set a component's background color to match the current HVAC mode (with a fall back option if the Item has some other state, e.g. `null`).
+For example, if there is an HVAC with a mode Item that can be set to `heat`, `cool`, `auto`, and `off` modes, it requires 4 nested ternary operators to set a component's background color to match the current HVAC mode (with a fallback option if the Item has some other state, e.g. `null`).
 
-```js
+```javascript
 background: =(@@hvacModeItem == 'heat')?'orange':(@@hvacModeItem == 'cool')?'blue':(@@hvacModeItem == 'auto')?'green':(@@hvacModeItem == 'off')?'white':'red'
 ```
 
 To use an object instead, simply create an object with keys for each of the Item's expected states, and give each key the desired output value.
 Referencing that object using the Item's state will return the desired value.
 Adding a simple `OR` statement to that expression will provide the fallback condition if the object reference is undefined.
 
-```js
+```javascript
 background: =({heat:'orange',cool:'blue',auto:'green',off:'white'})[@@hvacModeItem] || 'red'
 ```
 
 ## Examples
 
 Translates the third part of the HSB state (brightness) of an `Color` Item to "On" or "Off":
 
-```js
+```javascript
 =(@@'Color1'.split(',')[2] !== '0') ? 'On ' + '(' + @@'Color1'.split(',')[2] + '%)' : 'Off'
 ```
 
-Use a filled icon of a lightbulb but only if the state of the Item passed in the prop `item` is ON:
+Use a filled icon of a light bulb but only if the state of the Item passed in the prop `item` is ON:
```

Comment:
AFAIK lightbulb and light bulb both are fine. If you are referring to the name of the icon, though, I would keep it together.